data_steward/gcloud/bq/__init__.py

```diff
@@ -12,13 +13,16 @@
 from opentelemetry.exporter.cloud_trace import CloudTraceSpanExporter
 from opentelemetry.sdk.trace import TracerProvider
 from opentelemetry.sdk.trace.export import BatchSpanProcessor
+from google.api_core.exceptions import GoogleAPIError, BadRequest
 
 # Project imports
 from utils import auth
 from resources import fields_for
 from constants.utils import bq as consts
 from common import JINJA_ENV
 
+LOGGER = logging.getLogger(__name__)
```

Comment:
You mean OpenTelemetry?

Reply:
not the open telemetry, ou local logging implementation, will try to log to already existing logger that is initiated in the Bigquery library for which we do not have write access to, this will cause an issue when running inside of app-engine here is the bug ticket we filed when we did it for gcs - https://precisionmedicineinitiative.atlassian.net/browse/DC-2193 
Here is the PR for when we had to remove the local logging from gcloud/gcs - https://github.com/all-of-us/curation/commit/71f7c0a0c5ca23b696fff6455fc9345ccc93ec5a

Reply:
I see. So, we remove the logger?